src/instrumenter/annotations.ts

```diff
@@ -395,7 +395,7 @@ class AnnotationExtractor {
 
         const result: AnnotationMetaData[] = [];
 
-        const rx = /\s*(\*|\/\/\/)\s*(if_succeeds|if_updated|if_assigned|invariant|define\s*[a-zA-Z0-9_]*\s*\([^)]*\))/g;
+        const rx = /\s*(\*|\/\/\/)(\s*#)?\s*(if_succeeds|if_updated|if_assigned|invariant|define\s*[a-zA-Z0-9_]*\s*\([^)]*\))/g;
```

Comment:
I don't think we should allow space between the `#` and the keyword. (So no `#  if_succeeds`)

Reply:
Addressed in https://github.com/ConsenSys/scribble/pull/44/commits/02b98f0d598cab8ac5fc53764fc058c3f88404a8